rust_src/src/crypto/mod.rs

```diff
@@ -106,16 +106,14 @@ fn get_coding_system_for_buffer(
     if LispObject::from(unsafe { globals.f_Vcoding_system_for_write }).is_not_nil() {
         return LispObject::from(unsafe { globals.f_Vcoding_system_for_write });
     }
-    if LispObject::from(buffer.buffer_file_coding_system).is_nil() || LispObject::from(unsafe {
+    if (LispObject::from(buffer.buffer_file_coding_system).is_nil() || LispObject::from(unsafe {
```

Comment:
I would actually pull out this whole condition.

Reply:
Pull out and do what with it? Do you mean make some local variables?

Reply:
Yep.